daemons/ipa-kdb/ipa_kdb_principals.c

```diff
@@ -1483,9 +1515,6 @@ krb5_error_code ipadb_get_principal(krb5_context kcontext,
     }
 
     if (!is_request_for_us(kcontext, ipactx->local_tgs, search_for)) {
-        krb5_klog_syslog(LOG_INFO,
-                         "ipadb_get_principal: requested principal "
-                         "is not for our realm\n");
         return KRB5_KDB_NOENTRY;
```

Comment:
question: was this spamming the logs unnecessarily?

Reply:
Yes. We added this as part of server referrals code because we thought it would be good to know about such cases. Apparently, every time you kinit as AD user, you get this message and it serves no real purpose other than scaring potential sysadmins who would think this is something wrong. So I discussed this with Isaac and decided to remove it.